src/openrct2-ui/windows/TrackDesignManage.cpp

```diff
@@ -235,7 +233,8 @@ namespace OpenRCT2::Ui::Windows
         auto ft = Formatter();
         ft.Add<const utf8*>(_trackDesignFileReference->name.c_str());
         DrawTextWrapped(
-            rt, { windowPos.x + (WW_DELETE_PROMPT / 2), windowPos.y + ((WH_DELETE_PROMPT / 2) - 9) }, (WW_DELETE_PROMPT - 4),
-            STR_ARE_YOU_SURE_YOU_WANT_TO_PERMANENTLY_DELETE_TRACK, ft, { TextAlignment::CENTRE });
+            rt, { windowPos.x + (kWindowSizeDeletePrompt.width / 2), windowPos.y + ((kWindowSizeDeletePrompt.height / 2) - 9) },
```

Comment:
Can now be simplified to just `windowPos + { ... }`. That'll hopefully help readability.